ios/swift/AmpliSwiftSampleApp/Shared/AmpliSwiftSampleAppApp.swift

```diff
@@ -41,8 +42,13 @@ struct AmpliSwiftSampleAppApp: App {
         let extraDict = ["test" : "extra test"];
 
         // Load
-        ampli.load(LoadOptions(client: LoadClientOptions(apiKey: apiKey)));
-
+        ampli.load(LoadOptions(client: LoadClientOptions(apiKey: apiKey)))
+        
+        // Add Middleware
+        ampli.amplitude?.addEventMiddleware(AMPBlockMiddleware { (payload, next) in
+            print(String(format:"[ampli] event=\(payload.event) payload=\(payload.extra)"))
```

Comment:
Do we want to call `next` in this example?

Reply:
Good catch